projects/plugins/jetpack/readme.txt

```diff
@@ -242,16 +242,22 @@ Jetpack Backup can do a full website migration to a new host, migrate theme file
 4. Promote your newest posts, pages, and products across your social media channels.
 
 == Changelog ==
-### 11.3-a.3 - 2022-08-09
+### 11.3-a.5 - 2022-08-16
 #### Enhancements
-- Payments block: use Block API v2 to simplify the overall markup.
-- Payments block: ensure the plan name field will update according to the other options selected, unless it's already been modified.
+- Payments block: add a new `useWidth` hook to control the width settings of a block.
+- Google Analytics: add support for the DNT sent by the browser.
+- Dashboard: add new card to highlight options available to WordPress agencies.
 
 #### Improved compatibility
-- Backup: add disclaimer text and link to the backup product card that links to an FAQ on the Pricing page.
+- VideoPress: migrate code from the plugin to the package.
+- VideoPress: move dependencies to the package.
```

Comment:
Same as with the changelog, though as there is no 'other' here they could just go.

Reply:
Resolved in [`62f9de3`](https://github.com/Automattic/jetpack/pull/25649/commits/62f9de3dbcf99df3c6538b28c624203120ad3966)